components/uniset/src/props.rs

```diff
@@ -30,373 +31,1772 @@ fn get_prop<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(
     ppucd_property.try_into()
 }
 
-/// Upper
-pub fn upper<'d, D: DataProvider<'d, UnicodeProperty<'d>> + ?Sized>(
+//
```

Comment:
Optional: move this file to *props/mod.rs* and make a custom *rustfmt.toml* that increases line length so that these match statements don't wrap weirdly.

Reply:
Done.